app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[0] == "cp" and len(command_list) == 3:
+        if command_list[1] != command_list[2]:
+            with (
+                open(command_list[1], "r") as input_file,
+                open(command_list[2], "w") as output_file
+            ):
```

Comment:
It's good that you're using a `with` statement for file handling, as it ensures that files are properly closed after their suite finishes. However, you should consider adding error handling to manage cases where the file paths might be invalid or the files might not be accessible. This can be done using a try-except block around the file operations.